src/OpenAPM.ts

```diff
@@ -169,7 +180,7 @@ export class OpenAPM extends LevitateEvents {
       );
 
       const metricReader = new PeriodicExportingMetricReader({
-        exporter: new ConsoleMetricExporter(),
+        exporter: new OTLPMetricExporter(this.otlpMetricExporterOptions),
         // Default is 60000ms (60 seconds). Set to 10 seconds for demonstrative purposes only.
         exportIntervalMillis: 10000
```

Comment:
`exportTimeoutMillis: process.env.OTLP_EXPORT_TIMEOUT || 60` @chtushar can you make this change?